x-pack/plugins/enterprise_search/public/applications/workplace_search/views/settings/settings_router.tsx

```diff
@@ -48,6 +46,9 @@ export const SettingsRouter: React.FC = () => {
           <SourceConfig sourceIndex={i} />
         </Route>
       ))}
+      <Route>
+        <Redirect to={ORG_SETTINGS_CUSTOMIZE_PATH} />
+      </Route>
```

Comment:
Screencap showing fallthrough/redirect behavior:

![404](https://user-images.githubusercontent.com/549407/123198465-00309700-d462-11eb-9168-2523a4f20718.gif)

IDK if we want 404s on every router eventually but this is mostly just for 7.14 FF expedience/time